modules/apps/analytics/analytics-client-js/src/utils/storage_version.js

```diff
@@ -28,10 +27,6 @@ const upgradeStorageSteps = [
 			if (storedContextKvArr && !Array.isArray(storedContextKvArr[0])) {
 				const storedContexts = new Map();
 
-				storedContextKvArr.forEach((context) => {
-					return storedContexts.set(legacyHash(context), context);
-				});
-
 				setContexts(storedContexts);
```

Comment:
Do we even care about `storedContexts` anymore?  It's just an empty map now.

Reply:
It's necessary to remove the `legacyHash` dependency, in this case we are not going to be able "convert" the old contexts any more because we're removing the old lib that was used to generate the hash. But we still need to convert it from array to map to avoid errors further.
IMHO I think it's ok in that case since we're already 9 months with this upgrade available in production.